### ISSUE TITLE
Add Creative Commons licenses

### ISSUE THREAD

**Issue by Feld0** [Poniverse/Pony.fm#50]
Creative Commons licenses, along with an explicit "broadcast this on pony radios" option, can achieve the same goals as Pony.fm's existing licenses but in a more legally sound fashion. The existing licenses should be deprecated in favour of CC options.

**Comment by jack126guy** [comment]
[Finally!](http://mlpforums.com/topic/58130-license-information/)

Anyway, I'd like to offer some suggestions to make this less legally dubious:

* In the discussion I linked, you say that CC licenses should grant broadcast permission automatically, which makes sense. In that case, I think three options (Pony.fm only, broadcast only, and CC license) would be easier to understand.
* Also in that discussion, you raised some concerns about CC licenses that allow commercial use. If this "deprecation" means changing from the current licenses to "equivalent" CC licences, then the equivalences should be so:
    * *Personal*: Pony.fm only
    * *Broadcast*: Broadcast only
    * *Open*: CC BY-NC-ND
    * *Remix*: CC BY-NC

* There should be a transition period to allow people to change to a different CC license or remove their tracks entirely. However, this means maintaining both sets of licenses for a while.

* [Pick a version and link to the actual licenses!](http://github.com/eqbeats/eqbeats/pull/20)

**Comment by Feld0** [comment]
@jack126guy Thanks for the comment. I'm considering revisiting whether broadcasting permission should even be an option, since it seems like a good thing for the pony community for as much music as possible to be available on radios. Besides, it's a way of drawing more attention to someone's music and that's a good thing for Pony.fm and Poniverse to be doing, right?

All CC licenses allow free sharing and redistribution of the content so a separate broadcast permission won't actually make sense if only CC licenses are available. The question becomes whether a non-CC option should exist. Thoughts?

Re-licensing content retroactively is legally dubious, as users haven't explicitly granted us the right to re-license their content, so I'd rather not do it. "Deprecation" will mean maintaining and honouring the existing licenses indefinitely but removing the ability to set them.

Whether commercial or non-commercial use should be permitted should be up to the artist and, in hindsight, it's not Pony.fm's call to make. Hypothetically, even if Pony.fm had the ability to sell music, it would be able to do so without a commercial CC license - as the artist has to specifically allow Pony.fm to be involved in the transaction for Pony.fm to be able to sell it on the artist's behalf.

I'm inclined to offer the entire spectrum of CC licenses with brief descriptions and links to each.

**Comment by jack126guy** [comment]
> "Deprecation" will mean maintaining and honouring the existing licenses indefinitely but removing the ability to set them.

Ah, I see. That sounds like a better idea. I think we should actively encourage people switch to a CC license as well.

Regarding a non-CC option... It *is* pretty standard on sites with CC licenses (SoundCloud, YouTube, Flickr, Bandcamp). Some people do want to sell their music and not let people download it for free. Then again, I would *love* to have a site that had only CC licenses. In this case I'd defer to the community. Maybe a poll on the forums?

By the way, if a non-CC option is kept, there's also the issue of downloads. A CC license should imply allowing downloads as well, so now we have broadcast, download, and CC. I think it should be laid out like this:

* No CC license
** Allow downloads?
** Allow broadcasting?
* CC license
** [Choose a license. I think this should be modeled after CC's [license chooser](http://creativecommons.org/choose/).]

Oh, and another thing: Should we also support [CC0](http://creativecommons.org/about/cc0/)? It's from Creative Commons, but it's not part of their regular license repertoire.

**Comment by Feld0** [comment]
Yeah, not offering a more restrictive license option would be a bad thing for those who sell downloads. 481 tracks currently have downloads disabled, which isn't an insignificant number.

A poll on MLP Forums won't be representative of the entire community; a non-CC option will be necessary as long as the ability to disable downloads is present, if anything.

I'd offer the CC0 option, too - dedicating your work to the public domain should be possible.

How about this?

### Pony.fm license
  - "is downloadable?" option is available and turned on by default
  - automatically allows broadcasting by our affiliates (but no offering a separate download); this can be revisited if/when someone complains about their song coming up on radio somewhere

### Creative Commons license
- "is downloadable?" option is greyed out and always selected
- license chooser for picking the precise license

### CC0/Public Domain
- "is downloadable?" option is greyed out and always selected

Thoughts on this, as well as which license to select as the default? The above brings us to eight options - twice as many as the current ones - and there will be artists who won't want to think too hard about licensing.

**Comment by jack126guy** [comment]
Okay, if the non-CC option is staying and there's not going to be an option to forbid broadcasting, then I think the options should be arranged like this:

* Pony.fm (and affiliates) only
* Download only
* CC license (allows downloads [1])
* Public domain/CC0 (allows downloads [1])

And then make it clear that uploading to Pony.fm automatically grants broadcast permission.

[1] These should be explicit statements, not simply a graying-out of checkboxes. I think grayed-out checkboxes are unintuitive; IMO they should be in a separate section, and this whole section should be grayed out or hidden.

**Comment by jack126guy** [comment]
As for the default license... Well, I think it should be the most conservative first, lest people complain about how people are using their music without their explicit permission.

You're right, people don't pay much attention to licensing, but I think there can be an appeal to sharing ("Let people share and remix your song"). This would have to somehow span the last two options.
